src/test/java/org/opensearch/security/dlic/rest/api/AccountApiTest.java

```diff
@@ -197,15 +200,15 @@ public void testPutAccountRetainsAccountInformation() throws Exception {
         final String testPassword = "test-password";
         final String newPassword = "new-password";
         final String createInternalUserPayload = "{\n" +
-                "  \"password\": \"" + testPassword + "\",\n" +
-                "  \"backend_roles\": [\"test-backend-role-1\"],\n" +
-                "  \"opendistro_security_roles\": [\"opendistro_security_all_access\"],\n" +
-                "  \"attributes\": {\n" +
-                "    \"attribute1\": \"value1\"\n" +
-                "  }\n" +
-                "}";
+		"  \"password\": \"" + testPassword + "\",\n" +
```

Comment:
Why is this change necessary?

Reply:
Revert back to the original formatting prior to PR #1172.

Reply:
Done